### ISSUE TITLE
Typos and Suggested Wording

### ISSUE THREAD

**Issue by jamiere-msft** [Microsoft/sqlworkshops#36]
Overall, the usage of "." after sentences is inconsistent.  Section 1.0 had most sentences ending with a period while 0.0 did not. 

**In the README.MD file**
- The second link under About this Workshop has the word "Architecture" misspelled
- "This README.MD file explains how the workshop is laid out, what you will learn" might be better phrased "This README.MD file explains how the workshop is structured, what you will learn"
0.0 Prerequisites
- "It does have an assumption to know basics of using a Linux bash shell but all commands are provided to run the activities including scripts." could be rephrased as "It does assume basic knowledge of using a Linux bash shell, though all commands are provided to run the activities including scripts."
- In the activities list, why does it have both bullets and numbers?  Maybe indent the 1-6 software required steps and use a different bullet.

**Activity:Prerequisites**
- The title "Activity: Prerequsites" had "Prerequisites" misspelled.
- Step #2 - rephrase to Login to your OpenShift cluster, via a web browser, using the URL provided to you for the openshiftConsoleUrl.
- Step #4 - add a ":" after "following"
- Step #6 - add a ":" after "following"
- Step #7  - add a ":" after "following"
- Step #8 - add a ":" after "like this"
1.0 SQL Server Deployment on OpenShift
- In the 3rd paragraph, second sentence, capitalize the "Y" in yaml.  It almost reads as oc.yaml with the capital Y
- in the 4th paragraph, place commas around "and others in the workshop" and add a ":" after "command like"

**Activity: Deploy SQL Server on OpenShift**
- Step #3 - add a ":" after "with sudo)"
- Step #4 - add a ":" after "script"
- Step #4 - add a ":" after "shell prompt"
- Step #5  - add a ":" after "shell prompt"
- Step #6 - add a ":" after "shell prompt"
- Step #7 - add a ":" after "shell prompt"
- Step #7 - maybe rephrase the last paragraph to: "You have now submitted a deployment.  The deployment is a logical collection of objects ..."
- Step #7 - maybe rephrase the last sentence to: "Proceed to the next step to check whether the deployment was successful."
- Step #8 - Rephrase to "When the value of AVAILABLE becomes 1, the deployment was successful and your container is running."
- Step #8 - Rephrase to "To verify if the deployment was successful, the STATUS of the pod should be Running and the LoadBalancer service has a valid IP address for EXTERNAL-IP."
- Step #9 - Rephrase to: "The SQL Server database engine produces a file called the ERRORLOG when it starts.  The ERRORLOG file can be used to gather interesting information about SQL Server and be used for troubleshooting."

**Comment by rgward** [comment]
Jamie, I addressed almost all of the issues here. In some cases, Buck had done a bit of wordsmith and reformatting which made the comment no longer applicable.
